interface/src/app/gene/network/network.js

```diff
@@ -187,20 +205,22 @@ angular.module('adage.gene.network', [
           NodeService.get(
             {'heavy_genes': heavyGenes, 'limit': 0},
             function success(response) {
-              var n = response.objects.length;
+              var i = 0, n = response.objects.length;
+              var uri;
               str += '<br />' + n + (n > 1 ? ' nodes are ' : ' node is ');
               str += 'related to both genes' + (n > 0 ? ':' : '.');
-              for (var i = 0; i < n; ++i) {
-                str += '<br />* ' + response.objects[i].name + '</li>';
+              for (; i < n; ++i) {
+                uri = '<a href="#/node/' + response.objects[i].id + '">';
```

Comment:
That is true. It is called `uri` mainly because in my original implementation it was acually an URI. After changing the code, I forgot to give it a more reasonable name. How about `anchorTag`?